recipes/nucdiff/meta.yaml

```diff
@@ -1,11 +1,11 @@
 package:
   name: nucdiff
-  version: "0.1.1"
+  version: "2.0.2"
 
 source:
-  fn: NucDiff-0.1.1.tar.gz
-  url: https://pypi.python.org/packages/c4/18/5dc9c833eba0278509b375fc69e11f19ef7a35288d623229f74730244ee8/NucDiff-0.1.1.tar.gz
-  md5: a318c1283ec1fb94ede6798588bfa804
+  fn: NucDiff-2.0.2.tar.gz
+  url: https://pypi.python.org/packages/49/bf/a83f96e9e0f5304038a2eeecfea74a10b5873dbd283d080b1a7b0e70d8fc/NucDiff-2.0.2.tar.gz
+  md5: b8c04b5353440784b82d9dce809c52ad
```

Comment:
Could you use
```yaml
  url: https://pypi.io/packages/source/N/NucDiff/NucDiff-2.0.2.tar.gz
  sha256: 6ee7b0a8780bbd305ec456f2b86fd8743d3ca475a92b77e599dbabbbb097e418
```
instead?

Reply:
of course; is sha256 the standard for bioconda?  I used `conda skeleton pypi` to set this up.

Reply:
Unless `md5` is the only explicitly given checksum, yes `sha256` should be used, see http://bioconda.github.io/guidelines.html#hashes.
`conda skeleton pypi` in `conda-build 2.1.x` is a bit outdated. With `conda-build 3.x` it'll use `sha256`.